### DISCUSSION TITLE
Measuring temperature on a steel pipe

### DISCUSSION THREAD

**Discussion by Ferdisss** [firemodels/fds#12595] [Q&A]
Hi! 
I want to make a FDS code to do the following: 
I want to make a 0.1 m wide steel pipe with a certain thickness. I want a fire to burn in a certain distance away from the fire in an open space, and I want to observe the temperatures in the pipe in two points along the pipe. 

I have two questions: 
- In smokeview, I only get the box with one measuring point, and there is no fire when I run simulation. Does anyone know what is wrong with the code? 
- I am going to use the results I get of the temperatures of the beam for something else. The beam is 10 meters wide, but I changed it to 0.1m in FDS since it wouldn't run when it was 10m wide. The fire is also made equally small in FDS. Does anyone know if this is ok to do, or if the results of the temperature will be wrong to be used on a 10 m wide beam? 

This is my code: 
&HEAD CHID = 'Steel beam' /
&TIME T_END = 3 / Seconds

&MESH ID = 'MESH', IJK = 15,15,15, XB = 0,0.15,0,0.15,0,0.15/

%MISC TMPA = 25 / Intital temperature

&SURF ID = 'STEEL BEAM'
      COLOR = 'STEEL BLUE'
      THICKNESS = 0.005
      MATL_ID = 'STEEL'
      TMP_INNER = 25
/ 

&OBST ID = 'STEEL BEAM'
	XB = 0.075,0.075,0.025,0.125,0.04,0.04, 
	SURF_ID = 'STEEL BEAM' 
	/

&SURF ID = 'Fire'
      TMP_FRONT = 400
      COLOR = 'RED'
      HRRPUA = 500
      TAU_Q = -1.0 
/ TAU_Q = -1 means that it will rise quickly. 

&OBST ID = 'Fire source'
	XB = 0.015,0.015,0.072,0.078,0.04,0.04
	SURF_ID = 'Fire'
	/

&REAC FUEL = 'PROPANE'
	CO_YIELD = 0.05
	SOOT_YIELD = 0.01 /
	

&MATL ID ='STEEL'
      EMISSIVITY = 0.7
      DENSITY = 7700
      CONDUCTIVITY = 45
      SPECIFIC_HEAT = 0.46 /

&DEVC XYZ = 0.075,0.05,0.04 QUANTITY = 'TEMPERATURE' ID = 'Temperature el 1'
&DEVC XYZ = 0.075,0.10,0.04 QUANTITY = 'TEMPERATURE' ID = 'Temperature el 2'

&TAIL /

**Comment by drjfloyd** [comment]
What is the total surface area of this OBST? What HRR do you get when you multiple the HRRPUA on Fire by that surface area?

&OBST ID = 'Fire source'
XB = 0.015,0.015,0.072,0.078,0.04,0.04
SURF_ID = 'Fire'
/

**Reply by Ferdisss** [reply]
Ah yes, I see now that I had zero distance in x-direction, and that the area therefore was zero. Thank you! 

I still don't see the steel pipe though.

![image](https://github.com/firemodels/fds/assets/160745378/f0eeb82f-6acb-4d23-9463-f5b93eacd285)

**Comment by drjfloyd** [comment]
Look carefully at the XB for your beam.

**Reply by Ferdisss** [reply]
Thank you! I was thinking that I only wanted 'a line' with a certain thickness. Do you know if it is possible to make a pipe with a thickness?
